build/examples.js

```diff
@@ -3,12 +3,6 @@ var path = require("path");
 module.exports = {
     mouseposition: path.join(__dirname, "..", "web", "client", "examples", "mouseposition", "app"),
     scalebar: path.join(__dirname, "..", "web", "client", "examples", "scalebar", "app"),
-    "3dviewer": path.join(__dirname, "..", "web", "client", "examples", "3dviewer", "app"),
     featuregrid: path.join(__dirname, "..", "web", "client", "examples", "featuregrid", "app"),
-    print: path.join(__dirname, "..", "web", "client", "examples", "print", "app"),
-    login: path.join(__dirname, "..", "web", "client", "examples", "login", "app"),
-    plugins: path.join(__dirname, "..", "web", "client", "examples", "plugins", "app"),
```

Comment:
API example uses the plugins example to store some information in the localStorage:

> Use the plugins example to save your configuration with a meaningful name and use it in the embedded example with the map=<name> parameter.

![image](https://user-images.githubusercontent.com/19175505/93102346-aec1d300-f6ab-11ea-8f66-9ab5acd9beef.png)

Reply:
Ok, so I think we should remove also that functionality, thanks.